### PR TITLE
Separate camera pan, rotation and zoom speed

### PR DESCRIPTION
**Pull request by MathemanFlo**
Armorbase part of PR to implement #1292.
I'm not 100% sure whether line 197 is correct. Should it be for panning?

### CODE REVIEW

**Comment by luboslenco** [comment]
> I'm not 100% sure whether line 197 is correct

It's for the Fly camera movement. ADQE keys for pan and WS keys for zoom, maybe we could split that as well.

File: Sources/arm/Config.hx [hunk 1]
```diff
@@ -33,6 +33,15 @@ class Config {
...
 				raw = Json.parse(blob.toString());
+
+				// Update existing configs to use newly added configurations
+				if (raw.camera_pan_speed == null)
+					raw.camera_pan_speed = 1.0;
+				if (raw.camera_zoom_speed == null)
+					raw.camera_zoom_speed = 1.0;
+				if (raw.camera_rotation_speed == null)
+					raw.camera_rotation_speed = 1.0;
+
```
Comment (luboslenco) [inline Sources/arm/Config.hx]: After git pull the config should get auto-reset (https://github.com/armory3d/armorbase/blob/main/Sources/arm/Config.hx#L119) so there is no need for this yet. Will put effort into preserving the config after v1.0.

Comment (MathemanFlo) [reply Sources/arm/Config.hx]: Totally forgot about that. Thank you for the reminder. Will remove the ugly piece of code.

**Comment by MathemanFlo** [comment]
I also added speeds for Camera mode: Rotate. Previously this mode did not support any speed changes. Thank you for letting me work on this feature.

**Pull request merged**